docker-compose/archiver-docker-compose.yml

```diff
@@ -0,0 +1,138 @@
+#
+# Archiver docker compose file for HDB++ and MariaDB
+#
+# Defines:
+#   - hdbpp-es: HDB++ Event subscriber
+#   - hdbpp-cm: HDB++ Configuration Manager
+#   - configurehdbpp: Container to configure the HDB++ Archiver
+#   - archiver-dsconfig: archiver Dsconfig device to configure HDB++ devices (Event Subscriber and Configuration Manager TANGO devices) in tango database
+#   - maria-db: MariaDB container
+
+# Requires:
+#   - tango
+#   - MariaDB
+#
+version: '2'
+
+volumes:
+  archiverdb: {}
+
+services:
+  hdbpp-es:
+      image: nexus.engageska-portugal.pt/tango-example/archiver:latest
+      network_mode: ${NETWORK_MODE}
+      container_name: ${CONTAINER_NAME_PREFIX}hdbpp-es
+      depends_on:
+        - databaseds
+        - archiver-dsconfig
+        - maria-db
+      environment:
+        - TANGO_HOST=${TANGO_HOST}
+        - HdbManager=archiving/hdbpp/confmanager01
+        # TODO: FOR FUTURE REFERENCE
+        #- HDB_TYPE=timescaledb
+        #- HDB_NAME=hdb
+      command: >
+        sh -c "wait-for-it.sh ${TANGO_HOST} --timeout=30 --strict --
+                    find . -iname hdbppes-srv && ./usr/local/bin/hdbppes-srv 01 v5"
+# TODO: FOR FUTURE REFERENCE
+#        /bin/bash -c "
+#          echo HELLO!!!;
+#          sleep 20;
+#          wait-for-it.sh ${TANGO_HOST} --timeout=30 --strict --
+#               hdbppes-srv 01"
+
+  hdbpp-cm:
+      image: nexus.engageska-portugal.pt/tango-example/archiver:latest
+      network_mode: ${NETWORK_MODE}
+      container_name: ${CONTAINER_NAME_PREFIX}hdbpp-cm
+      depends_on:
+        - databaseds
+        - archiver-dsconfig
+        - maria-db
+      environment:
+        - TANGO_HOST=${TANGO_HOST}
+        - HdbManager=archiving/hdbpp/confmanager01
+        # TODO: FOR FUTURE REFERENCE
+        #- HDB_TYPE=timescaledb
+        #- HDB_NAME=hdb
+      command: >
+        sh -c "wait-for-it.sh ${TANGO_HOST} --timeout=30 --strict --
+                            find . -iname hdbppcm-srv && ./usr/local/bin/hdbppcm-srv 01 v5"
+# TODO: FOR FUTURE REFERENCE
+#        /bin/bash -c "
+#          echo HELLO!!!;
+#          sleep 20;
+#          wait-for-it.sh ${TANGO_HOST} --timeout=30 --strict --
+#               hdbppcm-srv 01"
+
+  configurehdbpp:
+      image: nexus.engageska-portugal.pt/ska-docker/ska-python-buildenv:0.1.0
+      network_mode: ${NETWORK_MODE}
+      container_name: ${CONTAINER_NAME_PREFIX}configurehdbpp
+      depends_on:
+        - databaseds
+        - archiver-dsconfig
+        - maria-db
+        - hdbpp-es
+        - hdbpp-cm
+      environment:
+        - TANGO_HOST=${TANGO_HOST}
+      volumes:
+        - ../tmcprototype/configure_hdbpp.py:/configure_hdbpp.py
+        - ../data/attribute_fqdn.txt:/attribute_fqdn.txt
+      command: >
+        /bin/bash -c "
+           sleep 25;
```

Comment:
This delay is added when we were thinking of Cassandra. Now we have moved to MariaDB. So do we need this delay now also?

Reply:
Delay is needed here for the MariaDB to be up and running.